database/insertData.js

```diff
@@ -13,7 +13,28 @@ const queries = [
           code: "TestRego"
           name: "Test -- General Registration"
           isLinear: false
-          startMessage: "## This is the general registration for feature showcase\\n- Proof of identity (Passport, Drivers license)\\n- Proof of your medical certification\\n- Drug ingredient list\\n- Product images\\n- Packging images"
+          startMessage: {
```

Comment:
Would this be a good place to use the `substring` evaluator from PR [#158](https://github.com/openmsupply/application-manager-server/pull/164)?

Reply:
Yes, I was planning to, but can't yet as that update has not been published to package yet. 

I'll need to do a new PR to update the version in package.JSON once the evaluator has been updated, and was planning to change these strings in that PR.

Reply:
Ok. No problem. I was just pointing out because it seems more redundant using the `substring` type